src/components/pages/doc/mobile-nav/mobile-nav.jsx

```diff
@@ -14,20 +14,28 @@ const MobileNav = ({ className, sidebar, currentSlug }) => {
   return (
     <nav className={clsx('relative', className)}>
       <select
-        className="w-full appearance-none border-2 border-black bg-white px-5 py-3"
+        className="w-full appearance-none border-2 border-black bg-white px-5 py-3 outline-none"
         value={currentSlug}
         onChange={handleChange}
       >
         <option value={false} disabled>
           Choose section
         </option>
         {sidebar &&
-          sidebar.map(({ title, items }, index) => (
-            <optgroup label={title} key={index}>
-              {items.map(({ title, slug }, index) => (
-                <option label={title} value={slug} key={index}>
-                  {title}
-                </option>
+          sidebar.map((sidebarItem, index) => (
+            <optgroup label={sidebarItem.title} key={index}>
+              {sidebarItem.items.map(({ title, slug, items }, index) => (
+                <Fragment key={index}>
+                  {items && items.length > 0 ? (
+                    items?.map(({ title: title2, slug }, index) => (
```

Comment:
We've made the necessary checks already, haven't we?
```suggestion
                    items.map(({ title: title2, slug }, index) => (
```